--- /usr/local/lib/python3.6/site-packages/pgadmin4/pgadmin.old/__init__.py
+++ /usr/local/lib/python3.6/site-packages/pgadmin4/pgadmin/__init__.py
@@ -209,11 +209,12 @@
     create_app_data_directory(config)

     # File logging
-    fh = logging.FileHandler(config.LOG_FILE, encoding='utf-8')
-    fh.setLevel(config.FILE_LOG_LEVEL)
-    fh.setFormatter(logging.Formatter(config.FILE_LOG_FORMAT))
-    app.logger.addHandler(fh)
-    logger.addHandler(fh)
+    if config.LOG_FILE != '/dev/stdout':
+        fh = logging.FileHandler(config.LOG_FILE, encoding='utf-8')
+        fh.setLevel(config.FILE_LOG_LEVEL)
+        fh.setFormatter(logging.Formatter(config.FILE_LOG_FORMAT))
+        app.logger.addHandler(fh)
+        logger.addHandler(fh)

     # Console logging
     ch = logging.StreamHandler()
